psl-parser/src/main/java/org/linqs/psl/parser/CommandLineLoader.java

```diff
@@ -386,23 +382,24 @@ private static CommandLine parseOptions(String[] args) {
         } catch (ParseException ex) {
             System.err.println("Command line error: " + ex.getMessage());
             getHelpFormatter().printHelp("psl", options, true);
-            System.exit(1);
         }
+
         if (commandLineOptions.hasOption(OPTION_HELP)) {
             initDefaultLogger();
             getHelpFormatter().printHelp("psl", options, true);
             return commandLineOptions;
         }
+
         if (commandLineOptions.hasOption(OPTION_VERSION)) {
             initDefaultLogger();
             System.out.println("PSL CLI Version " + Version.getFull());
```

Comment:
This is no longer just the CLI.

Reply:
So "PSL CommandLine Loader Version " instead of "PSL CLI Version " ?